targets/frontend/next-env.d.ts

```diff
@@ -0,0 +1,2 @@
+/// <reference types="next" />
+/// <reference types="next/types/global" />
```

Comment:
It's marked as mandatory in the next.js doc. 
https://nextjs.org/docs/basic-features/typescript

Why did you comment on these lines?

Reply:
I thought it was created by next. I will look to the format

Reply:
We have the same on cdtn project... indeed it's auto generated :)